src/libcollections/slice.rs

```diff
@@ -812,22 +812,22 @@ impl<T> SliceExt for [T] {
     }
 
     #[inline]
-    fn slice<'a>(&'a self, start: uint, end: uint) -> &'a [T] {
+    fn slice<'a>(&'a self, start: usize, end: usize) -> &'a [T] {
```

Comment:
I think these lifetimes (and the corresponding ones on similar methods) can be elided.